src/session-search.ts

```
@@ -0,0 +1,73 @@
+ * session matched on project name only).
+ */
+export const findPromptMatch = (
+  prompts: string[],
+  wordsLower: string[],
+): PromptMatch | null => {
+  for (let i = 0; i < prompts.length; i++) {
+    const lower = prompts[i].toLowerCase();
+    for (const w of wordsLower) {
+      const idx = lower.indexOf(w);
+      if (idx !== -1) {
+        return { promptIndex: i, snippet: extractSnippet(prompts[i], idx, w.length) };
```

ESLint

[error] Replace `·promptIndex:·i,·snippet:·extractSnippet(prompts[i],·idx,·w.length)` with `⏎··········promptIndex:·i,⏎··········snippet:·extractSnippet(prompts[i],·idx,·w.length),⏎·······`

(prettier/prettier)